components/page-header/index.jsx

```diff
@@ -94,14 +94,16 @@ const renderChildren = (h, prefixCls, children) => {
 const PageHeader = {
   name: 'APageHeader',
   props: PageHeaderProps,
-  inject: {
-    configProvider: { default: () => ConfigConsumerProps },
+  setup() {
+    return {
+      configProvider: inject('configProvider', ConfigConsumerProps),
+    };
   },
   render(h) {
     const { getPrefixCls, pageHeader } = this.configProvider;
     const props = getOptionProps(this);
     const { prefixCls: customizePrefixCls, breadcrumb } = props;
-    const footer = getComponentFromProp(this, 'footer');
+    const footer = getComponent(this, 'footer');
     const children = this.$slots.default;
```

Comment:
this.$slots.default && this.$slots.default()

Reply:
Sorry forget